src/bin/ts2date/main.rs

```diff
@@ -0,0 +1,37 @@
+use std::io::{BufRead, Write};
+use chrono::{DateTime, Utc};
+use regex::Regex;
+use anyhow::Result;
+use dfir_toolkit::common::UnixTimestamp;
+use dfir_toolkit::common::FancyParser;
+use cli::Cli;
+
+mod cli;
+
+
+fn main() -> Result<()> {
+    let cli = Cli::parse_cli();
+
+    let mut input = cli.input_file;
+    let mut output = cli.output_file;
+
+    let re = Regex::new(r"^(?P<lhs>.*?)(?P<ts>\d{10})(?P<rhs>.*)$").unwrap();
+    
+    for line in input.lock().lines() {
+        let content = match line {
+            Ok(line) => line,
+            Err(_) => panic!("content of input file need to be in UTF-8 (not in UTF-16)"),
```

Comment:
```suggestion
            Err(_) => bail!("content of input file need to be in UTF-8 (not in UTF-16)"),
```
because this does not stop the program, but instead escalates the error, so that it can be handled above